app/scripts/modules/amazon/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -73,6 +73,31 @@ module.exports = angular.module('spinnaker.aws.serverGroup.configure.service', [
         }
       };
 
+      command.blockDeviceMappingsSource = () => {
```

Comment:
Also want to verify this doesn't leak into the task being sent to Orca - probably isn't, but worth making sure.